tests/core/test_sync_monitor.py

```diff
@@ -649,8 +664,10 @@ async def _mockpoint_sleep():
     async with frozen_clock.real_clock_timeout():
         await alice_core.wait_idle_monitors()
 
-    # Just make sure the sync is done
+    # Bob must now retrieve alice's changes
```

Comment:
must or should ? if so the sync shouldn't be in the test only

Reply:
I've changed the wording